esmvaltool/config-references.yml

```
@@ -603,6 +607,7 @@ references:
   davini18: "Davini, P., Zenodo. doi:10.5281/zenodo.1237837, 2018"
   deangelis15nat: "DeAngelis, A. M. et al., Nature, doi:10.1038/nature15770, 2015."
   demora2018gmd: "de Mora et al., Geosci. Model Dev., doi:10.5194/gmd-11-4215-2018, 2018."
+  docquier2017: " Docquier, D., Massonnet, F., Barthélemy, A., Tandon, N. F., Lecomte, O., and Fichefet, T.: Relationships between Arctic sea ice drift and strength modelled by NEMO-LIM3.6, The Cryosphere, 11, 2829-2846, https://doi.org/10.5194/tc-11-2829-2017, 2017. "
```

```suggestion
  docquier2017: " Docquier et al., The Cryosphere, https://doi.org/10.5194/tc-11-2829-2017, 2017. "
```
@SarahAlidoost how does it work now with bibtex?
Should the author add it or is it generated automatically?

---

Sorry, I should have scrolled the page. The bibtex file is there.